classes/Customer.php

```diff
@@ -790,7 +790,8 @@ public function getSimpleAddressSql($idAddress = null, $idLang = null)
                         `id_lang` = ' . (int) $idLang . '
                         AND `id_customer` = ' . (int) $this->id . '
                         AND a.`deleted` = 0
-                        AND a.`active` = 1';
+                        AND a.`active` = 1
+                    ORDER BY a.`alias`';
```

Comment:
Maybe we should keep it lined up like the above statements? 😄 
```suggestion
                        ORDER BY a.`alias`';
```
this makes it a little easier to read

Reply:
The previous statement is 'AND', so I choose to align my 'ORDER BY' statement with the 'WHERE' statement. Are you sure I have to fix this?

Reply:
Oh! I did not open the PR wide so I did not see the WHERE was not lined up.

You're right this is better this way 👍